spec/integration/general_shopping_list_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe 'General Shopping List', type: :feature do
+  describe 'index page' do
+    before(:each) do
+      User.destroy_all
+      Food.destroy_all
+      Recipe.destroy_all
+      RecipeFood.destroy_all
+      user = User.create(name: 'user', email: 'user@gmail.com', password: 123_456)
+      food = Food.create(name: 'meat', measurement_unit: 'lib', price: 1, quantity: 3, user:)
+      recipe = Recipe.create(name: 'bulgogi', preparation_time: 2, cooking_time: 1, description: 'good', public: true,
+                             user:)
```

Comment:
- [ ] Oh, not again the same thing 🤦🏻‍♂️. Make sure you completed it by adding an argument to the value.